content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -95,65 +102,80 @@ The StatefulSet manifest, included below, creates a Cassandra ring that consists
 1. Update the StatefulSet if necessary.
 2. Create the Cassandra StatefulSet from the `cassandra-statefulset.yaml` file:
 
-       kubectl create -f cassandra-statefulset.yaml
+    ```bash
+    kubectl create -f cassandra-statefulset.yaml
+    ```
 
 {{< code file="cassandra/cassandra-statefulset.yaml" >}}
 
 ## Validating The Cassandra StatefulSet
 
 1. Get the Cassandra StatefulSet:
 
-       kubectl get statefulset cassandra
+    ```bash
```

Comment:
This and the following indentations are breaking the numbered list. Please see preview result.

Reply:
The changes to this doc were included by accident. I've removed these changes from this PR.